src/inmanta/server/server.py

```diff
@@ -116,14 +120,24 @@ def get_dependencies(self) -> List[str]:
 
         database_username = opt.db_username.get()
         database_password = opt.db_password.get()
-        await data.connect(database_host, database_port, opt.db_name.get(), database_username, database_password)
+        self._pool = await data.connect(database_host, database_port, opt.db_name.get(), database_username, database_password)
         LOGGER.info("Connected to PostgreSQL database %s on %s:%d", opt.db_name.get(), database_host, database_port)
 
     async def disconnect_database(self) -> None:
         """ Disconnect the database
         """
         await data.disconnect()
 
+    async def get_status(self) -> Dict[str, ArgumentTypes]:
+        """ Get the status of the database connection
+        """
+        return {
+            "connected": self._pool is not None,
+            "poolsize": self._pool._queue.qsize(),
```

Comment:
is this the pool size or the size of blocked requests out of the pool?  (both numbers are usefull)